spark/src/main/java/com/nflabs/zeppelin/spark/ZeppelinContext.java

```diff
@@ -66,6 +67,17 @@ public void load(String artifact, boolean recursive) throws Exception {
     dep.load(artifact, recursive, false);
   }
 
+  public void load(String artifact, scala.collection.Iterable<String> excludes)
+      throws Exception {
+    dep.load(artifact,
+        scala.collection.JavaConversions.asJavaCollection(excludes),
```

Comment:
may be it would be cleaner to 

```java
static import scala.collection.JavaConversions.*
...
dep.load(artifaict, asJavaCollection(excludes), true, false);
```